src/asm/lexer.c

```diff
@@ -22,7 +22,7 @@
 #include <unistd.h>
 
 #include "extern/utf8decoder.h"
-#include "platform.h" /* For `mmap` */
+#include "platform.h" /* For `ssize_t` */
```

Comment:
for ssize_t and mmap

Reply:
The file [itself](https://github.com/rednex/rgbds/pull/557/files#diff-5b127b0a9db154ec733b35def43cbe58R51) shims around `mmap`, since it's definitely not a 1:1 replacement, so it's [just `ssize_t`](https://github.com/rednex/rgbds/pull/557/files#diff-594a5a7310de8ff3a6fedbd3b418bd16).

Reply:
d'oof